lib/tasks/truncate_data.rake

```diff
@@ -0,0 +1,24 @@
+namespace :ofn do
+  namespace :data do
+    desc 'Truncate data'
+    task truncate: :environment do
+      guard_and_warn
```

Comment:
This implicit dependency will bite us in the ass someday. What about using a more generic and reusable solution like https://github.com/adamcooke/rails-safe-tasks instead of our own?

Reply:
yes, that would be better... I think I have spent way too much time on this PR...